tests/config/pytest_software.yml

```diff
@@ -0,0 +1,5 @@
+# FIXME Tests don't work because of bin/ scripts
```

Comment:
Is that still true?

Reply:
Just threw some stuff together, and yes it's still true, I think it needs to come from the nextflow level before we can start testing local modules that depend on bin/ scripts. Or make more heavy use of custom modules.

Reply:
https://github.com/nf-core/nascent/commit/0ff751ad7de9f167d8bf794a2c044db7c88e16bb